spec/graphql/mutations/therapists/create_therapist_spec.rb

```diff
@@ -0,0 +1,78 @@
+require 'rails_helper'
+
+RSpec.describe 'create therapist' do
+  it 'creates a therapists and returns the therapist details' do
+    @practice_1 = Practice.create!(name: "ABC therapy", website_url: "WWW.ABCD.com")
+
+    query = <<~GQL
+      mutation {
+        createTherapist(input: {
+          name: "Bingo McAllister",
+          address: "888 Your Mom's House Cul-de-Sac",
+          phoneNumber: "555-555-5555",
+          labels: ["Sliding Scale"],
```

Comment:
I'm still really curious about this labels array. How does the FE know what the labels are? If labels are added or deleted or changed, where is the single source of truth for this information?

Reply:
The array would be created on the fe in a different iteration. Currently the fe is just using them to check boxes so they are an agreed upon set value. But I think the values array would end up saving money in the long run in the be, if the fe stored the values as objects on their end. On the be we are just storing the preset values that the fe has determined. Ideally fe would be the source of truth and the be is just saving the relationship that is instantiated on the fe.

Reply:
I like that you're thinking about this and all the aspects of storing the data. It's definitely one of those discussions that happens often on dev teams. One thing to think about, that if the FE is the source of truth though with just a list of labels, what happens if the "ADD" label changes to "ADHD" or other labels are updated as conditions are understood more... then the FE would update that label, but then there would be therapists with the label of ADD and some with ADHD. How would this be reconciled? Ideally, the change would only need to happen in ONE place to ensure the integrity of the data. One way this can be implemented is my having a Labels table on the BE. There would then be a many to many relationship with the therpaists table, but then there is a clear source of truth and anytime a label gets changed, it has effectively been changed for all therpists associated with that label.  Just more things to think about. :)

Reply:
See and I think We could run an update to all therapists using AR to go through the code and search for the specific term and then edit them that way, so that the be is still the source of truth but there isn't a need for another table. Maybe adding in a method for a mass update would be something cool to add if time allows